SPECS/flatbuffers/flatbuffers.spec

```diff
@@ -0,0 +1,212 @@
+memory efficiency. It allows you to directly access serialized data without
+parsing/unpacking it first, while still having great forwards/backwards
+compatibility.}
+Summary:        Memory efficient serialization library
+Name:           flatbuffers
+Version:        2.0.8
+Release:        1%{?dist}
+# The entire source code is Apache-2.0. Even code from grpc, which is
+# BSD-3-Clause in its upstream, is intended to be Apache-2.0 in this project.
+# (Google is the copyright holder for both projects, so it can relicense at
+# will.) See https://github.com/google/flatbuffers/pull/7073.
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://google.github.io/flatbuffers
+Source0:        https://github.com/google/flatbuffers/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc-c++
+# The ninja backend should be slightly faster than make, with no disadvantages.
+BuildRequires:  ninja-build
+BuildRequires:  python3-devel
+BuildRequires:  python3-packaging
+BuildRequires:  python3-pip
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-wheel
+# From grpc/README.md:
+#
+#   NOTE: files in `src/` are shared with the GRPC project, and maintained
+#   there (any changes should be submitted to GRPC instead). These files are
+#   copied from GRPC, and work with both the Protobuf and FlatBuffers code
+#   generator.
+#
+# It’s not clearly documented which GPRC version is excerpted, but see
+# https://github.com/google/flatbuffers/pull/4305 for more details.
+#
+# It is not possible to unbundle this because private/internal APIs are used.
+Provides:       bundled(grpc)
+
+%description %{common_description}
+
+BuildArch:      noarch
+
+%package -n     python3-flatbuffers
+Summary:        FlatBuffers serialization format for Python
+Recommends:     python3dist(numpy)
+Provides:       flatbuffers-python3 = %{version}-%{release}
+Obsoletes:      flatbuffers-python3 < 2.0.0-6
+BuildArch:      noarch
+
+%description -n python3-flatbuffers %{common_description}
+
+This package contains the Python runtime library for use with the Flatbuffers
+serialization format.
+
+%prep
+%autosetup
+%{py3_shebang_fix} samples
+
+# flattests fails if out-of-source build directory is used
+# https://github.com/google/flatbuffers/issues/7282
+#
+# This downstream-only patch is not general; it makes assumptions about the
+# out-of-source build path and would break in-source builds.
+sed -r -i 's@//((include|monster)_test)@//../../tests/\1@g' \
+    tests/test.cpp tests/reflection_test.cpp
+
+
+%generate_buildrequires
+pushd python >/dev/null
+%pyproject_buildrequires
+popd >/dev/null
+
+
+%build
+# Needed for correct Python wheel version
+export VERSION='%{version}'
+%set_build_flags
+
+# %cmake -GNinja \
+#     -DCMAKE_BUILD_TYPE=Release \
+#     -DFLATBUFFERS_BUILD_SHAREDLIB=ON \
+#     -DFLATBUFFERS_BUILD_FLATLIB=OFF \
+#     -DFLATBUFFERS_BUILD_FLATC=ON
```

Comment:
Let's remove unused commands.

Also, since this is commented out, do we really need `BuildRequires:  ninja-build`? Could you check that and remove the dependency, if it's not needed?